containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -66,7 +66,7 @@
 
 namespace Kokkos {
 
-enum { UnorderedMapInvalidIndex = ~0u };
+enum : unsigned { UnorderedMapInvalidIndex = ~0u };
```

Comment:
Did you consider making this a static constexpr member variable?

Reply:
yes but I ran into trouble.